packages/admin/cms-admin/src/dashboard/widgets/LatestContentUpdatesDashboardWidget.tsx

```diff
@@ -15,59 +15,66 @@ type MinimalRow = {
 };
 
 export type LatestContentUpdatesDashboardWidgetProps<Row extends MinimalRow> = {
-    tableQuery: Pick<TableQueryProps, "api" | "loading" | "error"> & {
-        tableData?: Partial<ITableProps<Row>> & Pick<ITableProps<Row>, "data" | "totalCount">;
-    };
     getUrlFromPageTreeNode?: (row: Row) => string;
-};
+    rows: DataGridProps<Row>["rows"] | undefined;
+} & Pick<DataGridProps<Row>, "loading" | "error">;
 
 export const LatestContentUpdatesDashboardWidget = <Row extends MinimalRow>({
-    tableQuery: { tableData, api, loading, error },
     getUrlFromPageTreeNode,
+    rows = [],
+    loading,
+    error,
 }: LatestContentUpdatesDashboardWidgetProps<Row>) => {
     const intl = useIntl();
+    const columns: GridColDef<Row>[] = [
+        {
+            ...disableFieldOptions,
+            field: "name",
+            headerName: intl.formatMessage({ id: "dashboard.latestContentUpdates.name", defaultMessage: "Page Name" }),
+            flex: 1,
+        },
+        {
+            ...disableFieldOptions,
+            field: "updatedAt",
+            headerName: intl.formatMessage({ id: "dashboard.latestContentUpdates.updatedAt", defaultMessage: "Updated At" }),
+            type: "dateTime",
+            flex: 1,
+            renderCell: ({ row }) => (
+                <>
+                    <FormattedDate value={row.updatedAt} day="2-digit" month="2-digit" year="numeric" />
+                    {", "}
+                    <FormattedTime value={row.updatedAt} />
+                </>
+            ),
+        },
+        {
+            ...disableFieldOptions,
+            field: "jumpTo",
+            headerName: "",
+            width: 60,
+            renderCell: ({ row }) => {
+                if (!getUrlFromPageTreeNode) {
+                    return null;
+                }
```

Comment:
We could alternatively add `url?: string` to `MinimalRow` and add it to the rows via a `rows.map(row => ({...row, url: "..." })` call. What do you prefer? Also should we call it `editUrl` instead? Just url could be confusing as a page tree node also has an url?

Reply:
Yes, I like the idea of adding it to the row 👍🏼